#[tokio::main]
async fn main() -> Result<(), lambda_runtime::Error> {
    Ok(())
}
